src/components/resources/Resources.js

```diff
@@ -0,0 +1,5 @@
+import React from "./node_modules/react"
```

Comment:
We shouldn't need the relative path here—

```js
import React from 'react'
```

—if it isn't working outright, though, we may want to adjust the configuration to enable it!